packages/build/tests/plugins/tests.js

```diff
@@ -135,13 +135,15 @@ const loadEdgeHandlerBundle = async function({ outputDir, manifestPath }) {
 }
 
 const getEdgeHandlerBundlePath = function({ outputDir, manifestPath }) {
+  // eslint-disable-next-line node/global-require
   const { sha } = require(manifestPath)
```

Comment:
Edge handlers tests create the manifest file and bundle dynamically, so must be required dynamically.